arrow-core-data/src/main/kotlin/arrow/core/Option.kt

```diff
@@ -431,6 +431,14 @@ sealed class Option<out A> : OptionOf<A> {
         None
       }
 
+    @Deprecated(
+      "empty is deprecated, and will be removed in 0.13.0. Please use None instead.",
+      ReplaceWith(
+        "None",
+        "arrow.core.None"
+      ),
+      DeprecationLevel.WARNING
+    )
     fun <A> empty(): Option<A> = None
 
     val unit: Option<Unit> = Some(Unit)
```

Comment:
I think we also discussed removing the `unit` constructor method, didn't we?

Reply:
Yes, you're correct @franciscodr. I updated the PR. I checked against 0.11.0.3 and most of these were added after, so we could get away with removing them insteaad of deprecating 😄

Thanks for the reminder! 🕵️